README.md

```diff
@@ -4,15 +4,26 @@
         <img width="800" src="logo-light.svg" alt="pg_duckdb logo" />
     </picture>
 </p>
-
-0.3.0 release is here 🎉 Please [try](#installation) it out!
+<p align="center">
+	0.3.0 release is here 🎉<br />
+	Please <a href="#try-it-out">try</a> it out!
+</p>
 
 # pg_duckdb: Official Postgres extension for DuckDB
 
 pg_duckdb is a Postgres extension that embeds DuckDB's columnar-vectorized analytics engine and features into Postgres. We recommend using pg_duckdb to build high performance analytics and data-intensive applications.
 
 pg_duckdb was developed in collaboration with our partners, [Hydra][] and [MotherDuck][].
 
+## Try It Out
+
+The easiest way to try pg_duckdb is using the Hydra python package. Try now locally or deploy to the cloud:
+
+```
+pip install hydra-cli
+hydra
+```
```

Comment:
It seems to me like this would fit better as one of the subsections in the "Installation" section. Then we also don't have to change the at the top of the page.

Reply:
it's marketing, we'd like it up here for now, we can move it down or re-organize it later.